Automattic/jetpack
# Conversation/Dialogue: generic follow up tasks

In this PR we fix and polish issues and/or minor enhancements that couldn't be handled in previous PRs. Please feel free to add others in case you consider it appropriate.

* dialogue: add i18n context for label https://github.com/Automattic/jetpack/commit/618129cda02faabd4d056ad44314a71cc29e3701
* media-player-control: tabs > spaces https://github.com/Automattic/jetpack/commit/6225160825754d29d9dff45a706300ff0a8ab178
* Media Source: tidy and organize store
* Remove play and sync buttons. 420-gh-Automattic/dotcom-manage
No more additional controls:

<img src="https://user-images.githubusercontent.com/77539/105482171-c840e000-5c86-11eb-9a92-f2d85c75454c.png" width="500px" />



#### Changes proposed in this Pull Request:

* Conversation/Dialogue: generic follow up tasks

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Take a look at the PR description
* Try to check the fixes/changes
* Test the media source store, adding/removing/and playing with podcast players.
* Test the app in the front-end
* In general, the app should work as expected.

#### Proposed changelog entry for your changes:


n/a
